app/controllers/discourse_activity_pub/category_controller.rb

```diff
@@ -0,0 +1,88 @@
+# frozen_string_literal: true
+
+module DiscourseActivityPub
+  class CategoryController < ApplicationController
+    PAGE_SIZE = 50
+    ORDER = %w(actor user followed_at)
+
+    before_action :ensure_site_enabled
+    before_action :find_category
+    before_action :ensure_category_enabled
+
+    def index
+    end
+
+    def followers
+      guardian.ensure_can_see!(@category)
+
+      followers = @category
+        .activity_pub_followers
+        .joins(:follow_follows)
+        .where(follow_follows: { followed_id: @category.activity_pub_actor.id })
+        .joins(:user)
```

Comment:
```suggestion
        .left_joins(:user)
```

otherwise followers that aren't matched to Discourse users won't show up (found this out via a local test). We should accompany this with a test as well.

Reply:
Ah thanks for catching that. Fixed: https://github.com/angusmcleod/discourse-activity-pub/commit/7e24cfd4f2002eda55700c2f04f80ddad3988760